src/js/components/DoubleRangeSlider/DoubleRangeSlider.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import {
+  CFormRange, CFormInput, CFormLabel, CCol, CRow,
+} from '@coreui/react';
+import PropTypes from 'prop-types';
+import RangeRheostatGraph from '../Rheostat/RangeRheostatGraph';
+
+const DoubleRangeSlider = ({
+  resortList, sliderMin, sliderMax, scoreType, useGraph, tickerQuantity, className,
+}) => {
+  const [lowerValue, setLowerValue] = useState(50);
+  const [upperValue, setUpperValue] = useState(500);
+  const [lowerFieldValue, setLowerFieldValue] = useState(50);
+  const [upperFieldValue, setUpperFieldValue] = useState(500);
+
+  const widthSpanPercent = 100 / (sliderMax - sliderMin);
+  const leftPosition = (lowerValue - sliderMin) * widthSpanPercent;
+  const rightPosition = (upperValue - sliderMin) * widthSpanPercent;
+  const rangeWidth = (rightPosition + sliderMin) - (leftPosition + sliderMin);
+
+  function setLowerSlider(targetUpperValue) {
+    if ((lowerValue > targetUpperValue - 8) || (targetUpperValue < lowerValue + 8)) {
+      if (targetUpperValue >= sliderMax) {
+        setLowerValue(parseInt(sliderMax, 10) - 8);
+        setLowerFieldValue(parseInt(sliderMax, 10) - 8);
+      }
+      setLowerValue(Math.max(targetUpperValue - 8, sliderMin));
+      setLowerFieldValue(Math.max(targetUpperValue - 8, sliderMin));
+      if (targetUpperValue <= sliderMin + 8) {
+        setUpperValue(sliderMin + 8);
+        setUpperFieldValue(sliderMin + 8);
+      }
+    }
+  }
+
+  function setUpperSlider(targetLowerValue) {
+    if ((targetLowerValue > upperValue - 8) || (upperValue < targetLowerValue + 8)) {
+      if (targetLowerValue < sliderMax - 8) {
+        setUpperValue(targetLowerValue + 8);
+        setUpperFieldValue(targetLowerValue + 8);
+      }
+      setUpperValue(Math.min(targetLowerValue + 8, sliderMax));
+      setUpperFieldValue(Math.min(targetLowerValue + 8, sliderMax));
+      if (targetLowerValue >= sliderMax - 8) {
+        setLowerValue(sliderMax - 8);
+        setLowerFieldValue(sliderMax - 8);
+      }
+    }
+  }
+
+  function limitInput(target) {
+    if (target > sliderMax) {
+      return parseInt(sliderMax, 10);
+    } if (target < sliderMin) {
+      return parseInt(sliderMin, 10);
+    }
+    return parseInt(target, 10);
+  }
```

Comment:
It's a shame we don't have `lodash`, because https://www.geeksforgeeks.org/lodash-_-clamp-method/

This could also be a 1 liner:

```suggestion
  const clampInput = (target) => parseInt(Math.min(sliderMax, Math.max(sliderMin, target)), 0);
```